src/lib/CRSF/CRSF.cpp

```diff
@@ -293,6 +296,10 @@ void ICACHE_RAM_ATTR CRSF::sendTelemetryToTX(uint8_t *data)
 void ICACHE_RAM_ATTR CRSF::setSyncParams(uint32_t PacketInterval)
 {
     CRSF::RequestedRCpacketInterval = PacketInterval;
+    CRSF::OpenTXsyncOffset = 0;
+    CRSF::OpenTXsyncWindow = 0;
+    CRSF::OpenTXsyncWindowSize = (int32_t)(20000/CRSF::RequestedRCpacketInterval);
```

Comment:
This needs a max((int32_t)(20000/CRSF::RequestedRCpacketInterval), 1) since 25Hz mode would result in OpenTXsyncWindowSize=0 that would always result in a division by 0 in line 333.

Reply:
Good catch!

Reply:
Fixed and tested on BETAFPV 900Mhz module.